.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,18 @@
+<!--
+- Please fill in this template according to the PR you're about to submit
+- Features and deprecations must be submitted against the master branch
+- Bug fixes must be submitted against the lowest branch where they apply
+  (lowest branches are regularly merged to upper ones so they get the fixes too)
+-->
+
 | Q             | A
 | ------------- | ---
-| Branch?       | "master" for new features / 2.7, 2.8 or 3.1 for fixes
```

Comment:
No need for removing it actually. Having this as a comment (as for the top comment you added in this PR) allows to simply keep it, and just type the value in front of it.
Having the field blank by mistake is probably not worse than having the full placeholder displayed 😅